content/en/blog/_posts/2023-03-08-Updates-to-the-Auto-refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+title: Updates to the Auto-refreshing Official CVE Feed
+date: 2023-03-17
+slug: k8s-cve-feed-beta
+---
+
+**Authors**: Cailyn Edwards (Shopify), Pushkar Joglekar (Credit Karma), Mahé Tardy (Isovalent)
+
+Since launching the [Auto-refreshing Official CVE feed](/docs/reference/issues-security/official-cve-feed/) as an `alpha`
+feature in the 1.25 release, we have made significant improvements and updates. We are excited to announce the release of the
+`beta` version of the feed. This blog post will outline the feedback received, the changes made, and talk about what is
+planned for the `stable` release.
+
+
+## Feedback from end-users
+
+SIG Security received some feedback from end-users:
+- The JSON CVE Feed [did not comply](https://github.com/kubernetes/website/issues/36808)
+  with the [JSON Feed specification](https://www.jsonfeed.org/) as its name
+  would suggest.
+- The feed could also [support RSS](https://github.com/kubernetes/sig-security/issues/77)
+  in addition to JSON Feed format.
+- Some metadata fields could be added to indicate the freshness of
+  [the whole feed](https://github.com/kubernetes/sig-security/issues/72), or
+  [specific CVEs](https://github.com/kubernetes/sig-security/issues/63). Also
+  indicating [the Prow job](https://github.com/kubernetes/sig-security/issues/71)
+  that recently updated the feed. See more ideas directly on the
+  [the umbrella issue](https://github.com/kubernetes/sig-security/issues/1).
+- The feed markdown table on the website [should be ordered](https://github.com/kubernetes/sig-security/issues/73)
+  from the most recent to the least recently announced CVE.
+
+## Summary of changes
+
+In response, the SIG [did a rework of the script generating the JSON feed](https://github.com/kubernetes/sig-security/pull/76)
+to comply with the JSON Feed specification from generation and add a
+`last_updated` root field to indicate overall freshness. This redesign needed a
+[corresponding fix on the Kubernetes website side](https://github.com/kubernetes/website/pull/38579)
+for the CVE feed page to continue to work with the new format.
+
+After that, [RSS feed support](https://github.com/kubernetes/website/pull/39513)
+could be added transparently so that end-users can consume the feed in their
+preferred format.
+
+Overall, the redesign based on the JSON Feed specification, which this time broke
+backward compatibility, will allow updates in the future to address the rest of
+the issue while being more transparent and less disruptive to end-users.
+
+### Updates
```

Comment:
```suggestion
### Progress since the alpha
```
?

Reply:
It's also things that are not yet address